lte/gateway/c/sctpd/src/sctpd.cpp

```diff
@@ -75,11 +80,27 @@ int signalHandler(int* end, std::unique_ptr<Server>& server,
   return 0;
 }
 
+static magma::mconfig::SctpD load_mconfig() {
+  magma::mconfig::SctpD mconfig;
+  if (!magma::load_service_mconfig_from_file(SCTPD_SERVICE, &mconfig)) {
+    mconfig.set_log_level(magma::orc8r::LogLevel::INFO);
+  }
+  return mconfig;
+}
+
 int main() {
   signalMask();
 
-  magma::init_logging("sctpd");
-  magma::set_verbosity(MDEBUG);
+  auto mconfig = load_mconfig();
+
+  magma::init_logging(SCTPD_SERVICE);
+  magma::set_verbosity(mconfig.log_level());
+
+  sentry_config_t sentry_config;
+  sentry_config.sample_rate = mconfig.sentry_config().sample_rate();
+  strncpy(sentry_config.url_native,
+          mconfig.sentry_config().dsn_native().c_str(), MAX_URL_LENGTH);
```

Comment:
seeing this in CI
```
lte/gateway/c/sctpd/src/sctpd.cpp: In function 'int main()':
lte/gateway/c/sctpd/src/sctpd.cpp:101:10: warning: 'char* strncpy(char*, const char*, size_t)' specified bound 255 equals destination size [-Wstringop-truncation]
  101 |   strncpy(sentry_config.url_native,
      |   ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
  102 |           mconfig.sentry_config().dsn_native().c_str(), MAX_URL_LENGTH);
```

Reply:
ah looks like same warning exists for sessiond? (would be great if you could address both 🙏 )

Reply:
I added a manual null termination but I am not sure if there is a default way how to do it in Magma? I saw that many people recommend `strlcpy` but it doesn't seem to be used in the project.